app/store/slices/search.js

```diff
@@ -0,0 +1,46 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+export const fetchWeather = createAsyncThunk(
+  'weather/fetchWeatherData',
+  async (city, thunkAPI) => {
+    try {
+      const response = await axios.get(`https://api.openweathermap.org/data/2.5/forecast?q=${city}&appid=1a8c344b0aaddd77f2a53a70916ddc5b&units=imperial`);
```

Comment:
recommend separating out appid as a variable

Reply:
I don't understand this?!?

Reply:
> I don't understand this?!?

like this
const appID = 1a8c344b0aaddd77f2a53a70916ddc5b
&appid=${appID}

Reply:
I support this comment